cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -222,6 +222,11 @@ parameters:
 - name: CS_SERVICE_PRINCIPAL_CREDS_BASE64
   description: Base64 encoded service principal credentials for CS. This is temporary for P1
   required: true
+- name: AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID
```

Comment:
You also need to add this to the makefile, see https://github.com/Azure/ARO-HCP/pull/554/files#diff-899bdfee21578ae2b7a8962419b05a9d271fad7d40a8f642ed20ff885ecaf052R34 as example.

Just note that once the change in CS is merged, we also need to update the cs image tag in the makefile too.

Reply:
added